src/commands/sgd/source/delta.ts

```diff
@@ -104,7 +108,9 @@ export default class SourceDeltaGenerate extends SfdxCommand {
       output.error = err.message
       process.exitCode = 1
     }
-    this.ux.log(JSON.stringify(output, null, 2))
+    if(output.success && !this.flags.silent){
+      this.ux.log(JSON.stringify(output, null, 2))
+    }
```

Comment:
**remarks:** I don't think we should gate the output with the success

In this case it means error will not be displayed anymore which is not what we want.